persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageEntity.kt

```diff
@@ -293,6 +300,11 @@ sealed class MessageEntityContent {
         val assetName: String,
     ) : Regular()
 
+    data class Composite(
+        val text: Text?,
+        val buttonList: List<ButtonEntity>
```

Comment:
For `Composite` in proto, `Item` is `oneof` so it's possible to not have any button, or is it somehow blocked in the composite message creation logic on each platform?

Reply:
Although possible, that should be unlikely.

A composite message without buttons would look like a regular text message, but without the possibilities to reply or react to it.

An empty list of buttons should be fine. When it bubbles up to the UI, the UI can just _not_ render buttons if there aren't any, so it's all good.